style.css

```diff
@@ -0,0 +1,698 @@
+  display: flex;
+  justify-content: space-evenly;
+}
+
+.alibaba {
+  width: 90px;
+  height: 90px;
+  color: silver;
+}
+
+/* Footer Section Mobile */
+.footer {
+  display: flex;
+  justify-content: space-evenly;
+  padding: 8vh 4vw;
+  font-size: 1em;
+  color: #272a31;
+  line-height: 140%;
+}
+
+.logo-footer {
+  width: 15vw;
+  height: 15vh;
+}
+
+.footer-logo {
+  display: flex;
+}
+
+.footer-logo-a {
+  display: flex;
+  text-decoration: none;
+  align-items: center;
+  color: #4a77e0;
+}
+
+.footer-logo-a h2 {
+  color: #272a31;
+}
+
+.fa-confluence {
+  color: #ec5242;
+}
+
+.footer-p {
+  padding-left: 6vw;
+}
+
+.desktop-p {
+  display: none;
+}
+
+/* ***********************************************************************************************************/
+
+/* AboutUs Section Mobile */
+
+/* Header Content Mobile AboutUs */
+.header-content-about {
+  background-image: url(images/mainbackground.png);
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 3vh 3vw;
+}
+
+.header-content-about h5 {
+  color: #ec5242;
+  font-size: 20px;
+  font-weight: bold;
+  margin: 0;
+}
+
+.header-content-about h1 {
+  color: #ec5242;
+  font-family: 'COCOGOOSE', sans-serif;
+  align-self: center;
+  font-size: 25px;
+  text-align: center;
+}
+
+.header-paragraph-about {
+  display: flex;
+  background-color: white;
+  border: 1px solid #d3d3d3;
+  padding: 0.1vh 2vw;
+  text-align: center;
+  line-height: 140%;
+}
+
+.headear-about-contact {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  padding-top: 5vh;
+  line-height: 160%;
+}
+
+.headear-about-contact span {
+  text-align: center;
+}
+
+/* About-us Section Mobile AboutUs */
+.about-us {
+  display: flex;
+  flex-direction: column;
+  border-bottom: 1px solid #272a31;
+}
+
+.about-us-image {
+  display: flex;
+}
+
+.about-us-image img {
+  width: 100%;
+  height: auto;
+}
+
+.about-us-2 {
+  display: flex;
+  flex-direction: column;
+  padding: 2vh 3vw;
+  text-align: center;
+}
+
+.about-quest {
+  color: #ec5242;
+}
+
+/* Past Activities Section Mobile AboutUs */
+.past-activities {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  gap: 5vh;
+  padding: 5vh 5vw;
+}
+
+.past-activities p {
+  text-align: center;
+}
+
+.past-activities-section-container {
+  display: flex;
+  flex-direction: column;
+  gap: 5vh;
+}
+
+.past-activities-section {
+  justify-content: center;
+  align-items: center;
+  height: 35vh;
+  background-image: url(images/pastactivity1.png);
+}
+
+.past-activities-section-2 {
+  justify-content: center;
+  align-items: center;
+  background-image: url(images/pastactivity2.png);
+  height: 35vh;
+}
+
+.past-activities-desc {
+  display: flex;
+  flex-direction: column;
+  text-align: center;
+  color: #fff;
+}
+
+/************* Destop Version *************/
+@media screen and (min-width: 768px) {
+  /* Header Section Desktop */
+  .burger-menu {
+    display: none !important;
+  }
```

Comment:
- [ ] The previous reviewer mentioned it but you still have 2 appearances of `!important`. Using `!important` is bad practice and should be avoided because it makes debugging more difficult by breaking the natural cascading in your stylesheets.

Reply:
Alright, thank you!